pkg/daemon/daemon.go

```diff
@@ -515,3 +533,24 @@ func tryCreateUdevRule() error {
 	}
 	return nil
 }
+
+func (dn *Daemon) rebootDisabled() bool {
+
+	deploymentRes := schema.GroupVersionResource{
+		Group:    "sriovnetwork.openshift.io",
+		Version:  "v1",
+		Resource: "sriovoperatorconfigs",
+	}
+	sriovoperatorconfig, err := dn.dynamicclient.Resource(deploymentRes).Namespace("default").Get("default", metav1.GetOptions{})
```

Comment:
Also, fetching the value each time is not that efficient but since this does not happen often I think it's more than acceptable.

Reply:
rebootDisabled() is only invoked once when the daemon starts

Reply:
even better. Sorry I was thinking it was invoked every time it needed to be checked.